drivers/bt_ll_nrfxlib/flash/flash_ll_nrfxlib.c

```diff
@@ -34,7 +34,7 @@ static struct {
 	struct k_sem sync;
 	const void *data;
 	off_t addr;
-	u16_t len;
+	u32_t len;
 	u16_t prev_len;
```

Comment:
prev_len also needs to be an uint32_t

Reply:
Are you sure about it ? Can value returned by: `nrfx_nvmc_flash_page_size_get` overflow `u16_t`? Or I missed something else?

Reply:
You are right. It may not be needed. I would still prefer to have it as a uint32 for consistancy

Reply:
Ok, then I will change it (I initially wanted to save some memory if possible)